js_modules/dagit/packages/core/src/asset-graph/AssetGraphExplorer.tsx

```diff
@@ -401,52 +398,45 @@ export const AssetGraphExplorerWithData: React.FC<
-                  title={titleForLaunch(selectedGraphNodes, liveDataByNode)}
-                  preferredJobName={explorerPath.pipelineName}
-                  assets={launchGraphNodes.map((n) => n.definition)}
-                  upstreamAssetKeys={uniqBy(
-                    flatMap(launchGraphNodes.map((n) => n.definition.dependencyKeys)),
-                    (key) => JSON.stringify(key),
-                  ).filter(
-                    (key) =>
-                      !launchGraphNodes.some(
-                        (n) => JSON.stringify(n.assetKey) === JSON.stringify(key),
-                      ),
-                  )}
-                />
-              </Box>
-              {!props.pipelineSelector && <OmittedAssetsNotice assetKeys={props.allAssetKeys} />}
-            </Box>
-          )}
-          {setOptions && (
-            <QueryOverlay>
-              <GraphQueryInput
-                items={graphQueryItems}
-                value={explorerPath.opsQuery}
-                placeholder="Type an asset subset…"
-                onChange={(opsQuery) =>
-                  onChangeExplorerPath({...explorerPath, opsQuery}, 'replace')
-                }
-                popoverPosition="bottom-left"
+          <Box
+            flex={{direction: 'column', alignItems: 'flex-end', gap: 8}}
+            style={{position: 'absolute', right: 12, top: 12}}
+          >
+            <Box flex={{alignItems: 'center', gap: 12}}>
+              <QueryRefreshCountdown
+                refreshState={liveDataRefreshState}
+                dataDescription="materializations"
               />
-            </QueryOverlay>
-          )}
+
+              <LaunchAssetExecutionButton
+                title={titleForLaunch(selectedGraphNodes, liveDataByNode)}
+                preferredJobName={explorerPath.pipelineName}
+                assets={launchGraphNodes.map((n) => n.definition)}
+                upstreamAssetKeys={uniqBy(
+                  flatMap(launchGraphNodes.map((n) => n.definition.dependencyKeys)),
+                  (key) => JSON.stringify(key),
+                ).filter(
+                  (key) =>
+                    !launchGraphNodes.some(
+                      (n) => JSON.stringify(n.assetKey) === JSON.stringify(key),
+                    ),
+                )}
+              />
+            </Box>
+          </Box>
+          <QueryOverlay>
+            <GraphQueryInput
+              items={graphQueryItems}
+              value={explorerPath.opsQuery}
+              placeholder="Type an asset subset…"
+              onChange={(opsQuery) => onChangeExplorerPath({...explorerPath, opsQuery}, 'replace')}
+              popoverPosition="bottom-left"
+            />
+          </QueryOverlay>
         </>
       }
       second={
-        !options.enableSidebar ? null : selectedGraphNodes.length === 1 && selectedGraphNodes[0] ? (
+        selectedGraphNodes.length === 1 && selectedGraphNodes[0] ? (
```

Comment:
Removing `options.enableSidebar` because it was never set to false anywhere. The "Asset Details Lineage" tab which this option was built for no longer uses this component